halyard-config/halyard-config.gradle

```diff
@@ -2,4 +2,7 @@ dependencies {
   compile spinnaker.dependency('bootActuator')
   compile spinnaker.dependency('bootWeb')
   compile spinnaker.dependency('lombok')
+
+  compile 'io.fabric8:kubernetes-api:2.2.161'
+  compile 'com.netflix.spinnaker.clouddriver:clouddriver-kubernetes:1.388.0'
```

Comment:
No good reason, this should fix this: https://github.com/spinnaker/spinnaker-dependencies/pull/49